appeals/web/src/server/appeals/appeal-details/withdrawal/withdrawal.controller.js

```diff
@@ -0,0 +1,369 @@
+import { postWithdrawalRequest } from './withdrawal.service.js';
+import logger from '#lib/logger.js';
+import { objectContainsAllKeys } from '#lib/object-utilities.js';
+import { STAGE, DOCTYPE } from '@pins/appeals/constants/documents.js';
+import {
+	postDocumentUpload,
+	postUploadDocumentsCheckAndConfirm,
+	renderDocumentUpload
+} from '../../appeal-documents/appeal-documents.controller.js';
+import { addDocumentDetailsFormDataToFileUploadInfo } from '../../appeal-documents/appeal-documents.mapper.js';
+import { getDocumentRedactionStatuses } from '../../appeal-documents/appeal.documents.service.js';
+import {
+	checkAndConfirmPage,
+	dateWithdrawalRequestPage,
+	manageWithdrawalRequestFolderPage,
+	withdrawalConfirmationPage,
+	withdrawalDocumentRedactionStatusPage
+} from './withdrawal.mapper.js';
+
+/** @type {import('@pins/express').RequestHandler<Response>}  */
+export const getViewWithdrawalDocumentFolder = async (request, response) => {
+	const { errors, currentAppeal } = request;
+
+	if (!currentAppeal || currentAppeal.appealStatus !== 'withdrawn') {
```

Comment:
As mentioned in another comment, the appeal status should use the shared constant from the data model rather than being hardcoded - same for other instances in this file